crates/bevy_pbr/src/atmosphere/resources.rs

```diff
@@ -367,7 +372,11 @@ impl SpecializedRenderPipeline for RenderSkyBindGroupLayouts {
                     blend: Some(BlendState {
                         color: BlendComponent {
                             src_factor: BlendFactor::One,
+                            #[cfg(not(target_arch = "wasm32"))]
                             dst_factor: BlendFactor::Src1,
+                            // Dual source blending is not supported on Web
```

Comment:
Just WebGL2, no? Afaik WebGPU does support dual source blending.

Reply:
You might be right about that, I saw this article: https://developer.chrome.com/blog/new-in-webgpu-130#dual_source_blending

I did get an error last time I tested with Chrome, maybe it needs that special directive at the top? 
```
enable dual_source_blending;
```

Reply:
This is the error I get otherwise:
```
Features(DUAL_SOURCE_BLENDING) is not enabled for this backend
```

Reply:
https://github.com/gfx-rs/wgpu/blob/700a2e8ced7966d5a89c0635734e8c325397bbb6/CHANGELOG.md#webgpu-compliant-dual-source-blending-feature